stories/utils/Components/Button/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { PureComponent } from 'react';
```

Comment:
We see it often that some libraries that are used by `wix-style-react` also need some design system components for the documentation.
Maybe we can simply add all the relevant components to `wix-storybook-utils`.
It can be as minimal as HTML with some CSS, no need for too fancy styling or variations at the moment.
wdyt?